### ISSUE TITLE
PHP 8.1 support and conflict with `brainmaestro/composer-git-hook`

### ISSUE THREAD

**Issue by llaville** [overtrue/phplint#142]
I've noticed when trying to install a fresh copy (without composer.lock) of `overtrue/phplint` from branch 8.1, that we run in trouble; see following output : 

```
devilbox@php-8.1.6 in /shared/backups/github/my-phplint $ composer update
Loading composer repositories with package information
Info from https://repo.packagist.org: #StandWithUkraine
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires brainmaestro/composer-git-hooks ^2.8.5 -> satisfiable by brainmaestro/composer-git-hooks[v2.8.5].
    - brainmaestro/composer-git-hooks v2.8.5 requires symfony/console ^3.2 || ^4.0 || ^5.0 -> found symfony/console[v3.2.0-BETA1, ..., 3.4.x-dev, v4.0.0-BETA1, ..., 4.4.x-dev, v5.0.0-BETA1, ..., 5.4.x-dev] but it conflicts with your root composer.json require (^6.0).
```

Reason is that package [brainmaestro/composer-git-hooks](https://github.com/BrainMaestro/composer-git-hooks/blob/v2.8.5/composer.json#L18-L21) latest release 2.8.5 does not support yet `symfony/console` v6.

See origin of issue, from commit c5a9df9024995b568d565917a25c19b14922efbd

Fix is easy, and PR will follow !

**Comment by llaville** [comment]
That should solve also issue  #125

**Comment by llaville** [comment]
@overtrue PR #143 available for review/merge

**Comment by overtrue** [comment]
@llaville Are you interested in maintaining this project? I can add you as a maintainer or transfer the project to you.

**Comment by llaville** [comment]
@overtrue Thanks for this offer ! I'm not sure to have enough time in future months, but I accept to be a co-maintainer.

**Comment by overtrue** [comment]
@llaville Thank you very much.

**Comment by llaville** [comment]
This issue was fixed by commit 1f4d556dd7e2fe13dc288cf7848f0b94cd114d6a (merged), so this report may be closed
